etc/layout_viewer/README

```diff
@@ -0,0 +1,15 @@
+To generate a layout trace:
+
+```
+./mach run -d -Z trace-layout <url>
+```
+
+or
+
+```
+./servo --trace-layout <url>
```

Comment:
This doesn't seem to work. Should it be removed? Or changed to use `-Z trace-layout`?